drivers/watchdog/wdt_iwdg_stm32.c

```diff
@@ -126,6 +126,17 @@ static int iwdg_stm32_install_timeout(struct device *dev,
 	LL_IWDG_SetPrescaler(iwdg, prescaler);
 	LL_IWDG_SetReloadCounter(iwdg, reload);
 
+	/* Wait for the register to be updated */
+	while (LL_IWDG_IsReady(iwdg) != 0) {
+		/* Wait until WVU, RVU, PVU are reset before updating  */
```

Comment:
I think it should be "Wait until WVU, RVU, PVU are **set** before updating"

Reply:
Indeed, the loop waits for the LL_IWDG_IsReady function to return True, meaning that update is completed (The bits are set "to indicate that an update is ongoing")